seismicpro/src/refractor_velocity/refractor_velocity_field.py

```diff
@@ -104,5 +105,85 @@ def smooth(self, radius, min_refractor_points=10):
         return type(self)(smoothed_items, n_refractors=self.n_refractors, survey=self.survey,
                           is_geographic=self.is_geographic)
 
+    def dump(self, path, encoding="UTF-8", col_size=11):
+        """Save the RefractorVelocityField instance to a file.
+
+        File example:
+        SourceX   SourceY        t0        x1        v1        v2
+        1111100   2222220     50.00   1000.00   1500.00   2000.00
+        ...
+        1111200   2222240     60.00   1050.00   1550.00   1950.00
+
+        Parameters
+        ----------
+        path : str
+            Path to the file.
+        encoding : str, optional, defaults to "UTF-8"
+            File encoding.
+        col_size : int, defaults to 10
+            Size of each columns in file. `col_size` will be increased for coordinate columns if coordinate names
+            are longer.
+
+        Returns
+        -------
+        self : RefractorVelocityField
+            RefractorVelocityField unchanged.
+
+        Raises
+        ------
+        ValueError
+            If RefractorVelocityField is empty.
+        """
+        if self.is_empty:
+            raise ValueError("Field is empty. Could not dump empty field.")
+        new_col_size = max(col_size, max(len(name) for name in self.coords_cols) + 1)
+        values = np.array([list(item.params.values()) for coords, item in self.item_container.items()])
+
+        columns = list(self.coords_cols) + list(self.param_names)
+        cols_format = '{:>{new_col_size}}' * 2 + '{:>{col_size}}' * (len(columns) - 2)
+        cols_str = cols_format.format(*columns, new_col_size=new_col_size, col_size=col_size)
+
+        data = np.hstack((self.coords, values))
+        data_format = ('\n' + "{:>{new_col_size}.0f}" * 2 + '{:>{col_size}.2f}' * 2 * self.n_refractors) * data.shape[0]
+        data_str = data_format.format(*data.ravel(), new_col_size=new_col_size, col_size=col_size)
+
+        with open(path, 'w', encoding=encoding) as f:
+            f.write(cols_str + data_str)
+        return self
+
+    @classmethod
+    def load(cls, path, encoding="UTF-8"):
```

Comment:
```suggestion
    def from_file(cls, path, encoding="UTF-8"):
```
+ unify params with `__init__`